ui/apps/pixano/src/app.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="data:," />
```

Comment:
question: what is also an error if you delete this line?

Reply:
The error without this line is a `"GET /favicon.ico HTTP/1.1" 404 Not Found` in the deployed app.

I am guessing that without a specified icon, the HTML file will automatically look for a "favicon.ico" in the current folder.

The Svelte favicon still loads correctly with our without this line, this just removes the useless 404 message.